modules/force_rtl.py

```diff
@@ -30,7 +31,7 @@ def force_rtl(drone: dronekit.Vehicle) -> bool:
     rtl_command = generate_command.return_to_launch()
 
     # Change drone mode
-    drone.mode = dronekit.VehicleMode("RTL")
+    drone.mode = FlightController.set_flight_mode("RTL")
```

Comment:
would it be drone.set_flight_mode? since that is the flightcontroller instance passed in

Reply:
good point, lowk missed that